src/main/java/it/reply/orchestrator/dto/cmdb/ComputeService.java

```diff
@@ -39,15 +39,27 @@
   @JsonProperty("images")
   @NonNull
   @NotNull
-  @Builder.Default
+  //@Builder.Default
```

Comment:
Why did you remove it?

Reply:
Because the decoration has no meaning if the class does not have @Builder decoration

Reply:
ok I removed it for good (no point in leaving dead commented code)